DSharpPlus.UnifiedCommands/Message/Conditions/CooldownCondition.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Concurrent;
+
+namespace DSharpPlus.UnifiedCommands.Message.Conditions;
+
+public class CooldownCondition : IMessageCondition
+{
+    public static ConcurrentDictionary<string, DateTimeOffset> Cooldown { get; set; } =
+        new(); // VERY INEFFICIENT. CHANGE ASAP
+    // This will hold expired times forever.
```

Comment:
you could use a MemoryCache, the package is pulled in via my (at the time of writing still wip) rest PR